sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -193,43 +193,41 @@ provide both synchronous and asynchronous operations for analyzing input documen
-|"prebuilt-businessCard"| Text extraction and prebuilt fields and values pertaining to English business cards
-|"prebuilt-idDocument"| Text extraction and prebuilt fields and values pertaining to US driver licenses and international passports
-|"prebuilt-receipt"| Text extraction and prebuilt fields and values pertaining to English sales receipts
-|"{custom-model-id}"| Text extraction, selection marks, tables, labeled fields and values from your custom documents
-
-Sample code snippets to illustrate using a DocumentAnalysisClient [here][sample_readme]. 
-More information about analyzing documents, including supported features, locales, and document types can be found 
+| Model                   | Features                                                                                                 |
+|-------------------------|----------------------------------------------------------------------------------------------------------|
+| "prebuilt-layout"       | Text extraction, selection marks, tables                                                                 |
+| "prebuilt-document"     | Text extraction, selection marks, tables, key-value pairs, and entities                                  |
+| "prebuilt-invoice"      | Text extraction, selection marks, tables, and prebuilt fields and values related to English invoices     |
+| "prebuilt-businessCard" | Text extraction and prebuilt fields, and values related to English business cards                        |
+| "prebuilt-idDocument"   | Text extraction and prebuilt fields and values related to US driver licenses and international passports |
+| "prebuilt-receipt"      | Text extraction and prebuilt fields and values related to English sales receipts                         |
+| "{custom-model-id}"     | Text extraction, selection marks, tables, labeled fields, and values from your custom documents          |
+
+Sample code snippets to illustrate using a DocumentAnalysisClient [here][sample_readme].
+More information about analyzing documents, including supported features, locales, and document types can be found
 [here][fr-models].
 
 ### DocumentModelAdministrationClient
 The [DocumentModelAdministrationClient][document_model_admin_sync_client] and
 [DocumentModelAdministrationAsyncClient][document_model_admin_sync_client] provide both synchronous and asynchronous operations
-- Build custom document analysis models to analyze text content, fields and values found in your custom documents. See example [Build a model](#build-a-model).
-  A `DocumentModel` is returned indicating the document types that the model can analyze, and the fields and schemas it will extract in
+- Build custom document analysis models to analyze text content, fields, and values found in your custom documents. See example [Build a model](#build-a-model).
+  A `DocumentModel` is returned indicating the document types that the model can analyze, and the fields, and schemas it will extract in
```

Comment:
Nit
```suggestion
  A `DocumentModel` is returned indicating the document types that the model can analyze, along with the fields and schemas it will extract.
```